cumulusci/tasks/metadata_etl/layouts.py

```diff
@@ -68,3 +68,70 @@ def _create_new_related_list(self, metadata, api_name, related_list):
             elem.append("customButtons", button)
 
         elem.append("relatedList", text=related_list)
+
+
+class AddFields(MetadataSingleEntityTransformTask):
+    entity = "Layout"
+    task_options = {
+        "layout_section": {
+            "description": "Which layout section the fields should be added to, default is Information",
```

Comment:
Is this sensitive to localization? I.e., can this default fail in an org whose default language is Spanish?

Reply:
Even if user-specified, do we have the same language dependency? What would you think about an index-based approach - specify say the first or last section, without naming it, either as the primary target or as a fallback if the specified section does not exist?